http-clients/apache-client/src/main/java/software/amazon/awssdk/http/apache/ApacheHttpClient.java

```diff
@@ -581,6 +599,10 @@ private HostnameVerifier getHostNameVerifier(AttributeMap standardOptions) {
 
         private SSLContext getSslContext(AttributeMap standardOptions) {
             TrustManager[] trustManagers = null;
+            if (standardOptions.containsKey(SdkHttpConfigurationOption.TLS_TRUST_MANAGERS_PROVIDER)) {
```

Comment:
Add validation here (and other clients) that both this and trust all can't both be set?

Reply:
Done